readme.md

```diff
@@ -8,11 +8,15 @@ An open-source re-implementation of RollerCoaster Tycoon 2. A construction and m
 | **develop** | [![AppVeyor](https://ci.appveyor.com/api/projects/status/7efnemxhon6i5n34/branch/develop?svg=true)](https://ci.appveyor.com/project/IntelOrca/openrct2-ject9) | [![Travis CI](https://travis-ci.org/OpenRCT2/OpenRCT2.svg?branch=develop)](https://travis-ci.org/OpenRCT2/OpenRCT2) | [![OpenRCT2.org](https://img.shields.io/badge/develop-v0.0.8-blue.svg)](https://openrct2.org/downloads/develop/latest) |
 
 ### Chat
-English:<br />
-[![Gitter](https://img.shields.io/badge/gitter-general-blue.svg)](https://gitter.im/OpenRCT2/OpenRCT2/non-dev)<br />
-[![Gitter](https://img.shields.io/badge/gitter-development-yellowgreen.svg)](https://gitter.im/OpenRCT2/OpenRCT2)<br />
-Nederlands:<br />
-[![Gitter](https://img.shields.io/badge/gitter-general-blue.svg)](https://gitter.im/OpenRCT2/OpenRCT2/Nederlands)
+If you want to help *make* the game, join the Developer Channel for your Language.
+
+If you need help, or want to talk to the developers, or just want to stay up to date, join the Non Developer channels.
```

Comment:
If you need help, or want to talk to the developers, or just want to stay up to date, join the non-developer channels.